crates/debugger_ui/src/session/running/variable_list.rs

```diff
@@ -499,6 +584,7 @@ impl VariableList {
             menu.action("Copy Name", CopyVariableName.boxed_clone())
                 .action("Copy Value", CopyVariableValue.boxed_clone())
                 .action("Edit Value", EditVariable.boxed_clone())
+                .action("Add Watcher", AddWatcher.boxed_clone())
```

Comment:
We should go with a more uniform naming (I believe other products refer to this feature as `watches`, not `watchers`).
```suggestion
                .action("Add Watch", AddWatcher.boxed_clone())
```

Reply:
Yeah, I had that before, but liked the term watcher better as it's more clear what it does based on the word itself. But happy to change it back again.

Reply:
I'd err on the side of familarity with other products here tbh